os/board/rtl8730e/src/component/bluetooth/api/rtk_bt_power_control.c

```diff
@@ -75,18 +53,14 @@ static uint32_t rtk_bt_suspend(uint32_t expected_idle_time, void *param)
 
 	pmvdbg("[BT_PS] Enter rtk_bt_suspend\r\n");
 
+	system_enable_power_save();
```

Comment:
Is that enabling ble power save mode?
Then no need to call disabling? I mean no need to call it in pair?

Reply:
Hi @ziyik , could you help to reply this?

Reply:
This was added in the early stage to ensure Wifi LPS and IPS are enabled.
This is not needed anymore, Wifi module will control its IPS and LPS.
Edward has removed it.